modules/serverless-applications-checking-terminal-output.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * knative-serving/debugging-serverless-applications.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-applications-checking-terminal-output_{context}"]
+= Checking terminal output
+
+You can check your deploy command output to see whether deployment succeeded or not. If your deployment process was terminated, you should see an error message in the output that describes the reason why the deployment failed. This kind of failure is most likely due to either a misconfigured manifest or wrong command.
+
+.Procedure
+
+* Open the terminal output for your application. This is an example of an error that you might see:
```

Comment:
IMHO, the error in this example would come from a `oc apply` command and be seen on a client (like a laptop). So either update the sentence here to reflect that or use a log-example that could come from an application.

Reply:
@ReToCode Could you PTAL on whether it's good now?